.circleci/config.yml

```diff
@@ -21,6 +21,7 @@ references:
     GIT_AUTHOR_NAME: circleci-consul
     GIT_COMMITTER_NAME: circleci-consul
     S3_ARTIFACT_BUCKET:  consul-dev-artifacts
+    BASH_ENV: ./build-support/scripts/functions.sh
```

Comment:
This is a pretty neat and easy way to allow us to use some of the functions and scripts that are already present in the `build-support` directory!